redisinsight/ui/src/pages/rdi/pipeline-management/components/navigation/Navigation.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
+import { Col } from 'uiSrc/components/base/layout/flex'
+import { Theme } from 'uiSrc/components/base/theme/types'
+
+// TODO: move all props to the page wrapper
```

Comment:
This is migrated from the old styles, but in upcoming PR this is going to be a responsibility of the parent component.